src/server/routes/consents.ts

```diff
@@ -229,7 +244,7 @@ export const consentPages: ConsentPage[] = [
         page: 'review',
         consents: await getUserConsentsForPage(ALL_CONSENT, ip, sc_gu_u),
         newsletters: await getUserNewsletterSubscriptions(
-          NewsletterMap.get(geo) as string[],
+          TestNewsletterMap.get(geo) as string[],
```

Comment:
This change seems to fall outside the test, which may cause issues with reading newsletter subscriptions